check_tweets.py

```diff
@@ -0,0 +1,54 @@
+import mwclient
+import tweepy
+from datetime import date, timedelta
+import time
+from dotenv import load_dotenv
+import os
+import re
+
+load_dotenv()
+bearertoken = os.getenv("bearertoken")
+
+checkfrom = date.today() - timedelta(days=4)
+
+def main():
+	client = tweepy.Client(bearer_token=bearertoken)
```

Comment:
maybe call this `twitter_client` to avoid confusion with `site.client` as the wiki client